cache/src/test/kotlin/com/dropbox/android/external/cache4/CacheBuilderTest.kt

```diff
@@ -1,82 +1,85 @@
 package com.dropbox.android.external.cache4
 
 import com.dropbox.android.external.cache4.CacheBuilderImpl.Companion.DEFAULT_CONCURRENCY_LEVEL
-import com.dropbox.android.external.cache4.CacheBuilderImpl.Companion.DEFAULT_EXPIRATION_NANOS
 import com.dropbox.android.external.cache4.CacheBuilderImpl.Companion.UNSET_LONG
 import com.google.common.truth.Truth.assertThat
 import org.junit.Assert.assertThrows
 import org.junit.Test
-import java.util.concurrent.TimeUnit
+import kotlin.time.Duration
+import kotlin.time.ExperimentalTime
+import kotlin.time.hours
+import kotlin.time.nanoseconds
 
+@ExperimentalTime
 class CacheBuilderTest {
 
     @Test
     fun `set expireAfterWrite with zero duration`() {
         val cache = Cache.Builder.newBuilder()
-            .expireAfterWrite(0, TimeUnit.NANOSECONDS)
+            .expireAfterWrite(0.nanoseconds)
             .build<Any, Any>() as RealCache
 
-        assertThat(cache.expireAfterWriteNanos)
-            .isEqualTo(0)
+        assertThat(cache.expireAfterWriteDuration)
+            .isEqualTo(Duration.ZERO)
     }
 
     @Test
     fun `set expireAfterWrite with positive duration`() {
         val cache = Cache.Builder.newBuilder()
-            .expireAfterWrite(24, TimeUnit.HOURS)
+            .expireAfterWrite(24.hours)
             .build<Any, Any>() as RealCache
 
-        val expectedExpireAfterWriteNanos = 24 * 60 * 60 * 1000L * 1000L * 1000L
+        val expectedexpireAfterWriteDuration = 24 * 60 * 60 * 1000L * 1000L * 1000L
```

Comment:
```suggestion
        val expectedexpireAfterWriteDuration = 24.hours
```
I don't think we need to test with converted values now that we are using `kotlin.time.Duration` everywhere.